fun/management/commands/enroll_all_students.py

```diff
@@ -103,9 +100,10 @@ def handle(self, *args, **options):
                     diff = datetime.datetime.now(UTC) - start
                     timeleft = diff * (total_students - enrolled) / STATUS_INTERVAL
                     hours, remainder = divmod(timeleft.seconds, 3600)
-                    minutes, seconds = divmod(remainder, 60)
-                    print "{0} students out of {1} were successfully enrolled to course {2} ~{3:02}:{4:02}m remaining".format(
-                        enrolled, total_students,COURSE_ID, hours, minutes)
+                    minutes, _seconds = divmod(remainder, 60)
+                    print ("{0} students out of {1} were successfully "
+                           "enrolled to course {2} ~{3:02}:{4:02}m remaining").format(
+                        enrolled, total_students, COURSE_ID, hours, minutes)
 
```

Comment:
Une string sur plusieurs lignes est automatiquement concaténée. C'est joli et très pratique pour les longues strings. 
https://www.python.org/dev/peps/pep-0008/
"The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation."

C'est pas obligatoire pour nous mais je trouve ça pas mal.